hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapred/lib/MultipleOutputs.java

```diff
@@ -527,9 +557,42 @@ public void collect(Object key, Object value) throws IOException {
    * @throws java.io.IOException thrown if any of the MultipleOutput files
    *                             could not be closed properly.
    */
-  public void close() throws IOException {
+  public void close() throws IOException, InterruptedException {
```

Comment:
`InterruptedException` is not thrown in this method and should be removed. This class is annotated `@Public` and the change may cause compile error. Also, we can remove the below code from the test code.
```
      try {
        mos.close();
      } catch (InterruptedException e) {
        throw new RuntimeException(e);
      }
```

Reply:
makes sense. Addressed.